docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -0,0 +1,213 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:snip6: https://github.com/ericnordelo/SNIPs/blob/feat/standard-account/SNIPS/snip-6.md[SNIP-6]
+
+= Account
+
+Reference of interfaces, presets, and utilities related to Account contracts.
+
+== Core
+
+[.contract]
+[[ISRC6]]
+=== `++ISRC6++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/account/interface.cairo#L12[{github-icon},role=heading-link]
+
+[.hljs-theme-light.nopadding]
+```javascript
+use openzeppelin::account::interface::ISRC6;
+```
+
+Interface of the ISRC6 standard as defined in the {snip6}.
```

Comment:
```suggestion
Interface of the SRC6 Standard Account Interface as defined in {snip6}.
```
- the standard is SRC, not ISRC
- i lean towards stating what SRCs are when we reference them (e.g. call it "SRC6 standard account interface")

Reply:
> the standard is SRC, not ISRC

+1

> lean towards stating what SRCs are when we reference them (e.g. call it "SRC6 standard account interface")

To me, this would just add unnecessary repetition. We may reference the SNIPs a lot, and people can always get to the SNIP to get more context if they don't have it already. In Solidity we don't add the name for each EIP reference.

Also, "Interface of the SRC6 Standard Account Interface" is a weird sentence imo.

Reply:
I like that last one